packages/cozy-client/src/CozyClient.spec.js

```diff
@@ -527,6 +527,34 @@ describe('CozyClient', () => {
       ])
     })
 
+    it('should handle associations for a new file with relationship', () => {
+      const NEW_FILE = {
+        _type: 'io.cozy.files',
+        icons: [
```

Comment:
Can you add a comment expliciting that icons has been set as a HasMany relationship ? It is easy to forgot that icomes from the fixtures file.